Automattic/woocommerce-payments
# Disable new skeleton loader experience

Fixes #4327

#### Changes proposed in this Pull Request
Adds loader: 'never' to Payment Element integration in our codebase in order to disable the Stripe skeleton loader. This code change should be deployed before Stripe releases the Payment Element UI change, scheduled for July 12.

#### Testing instructions
Test that the loader does not appear and no ill-effects observed:
- Checkout
- Blocks Checkout
- Subscription Payment
- Any other pages showing payment form.
- Test on different [popular themes](https://href.li/?https://storeleads.app/reports/woocommerce#themes)

-------------------


**Post merge**

- [x] Link to testing instructions from [release testing doc](https://github.com/Automattic/woocommerce-payments/wiki/Release-testing-instructions) following [these instructions](https://github.com/Automattic/woocommerce-payments/wiki/How-to-write-good-manual-testing-scenarios) : [_Add link here / 'QA Testing Not Applicable'_](https://github.com/Automattic/woocommerce-payments/wiki/Release-testing-instructions-for-WC-Payments-4.4.0#test-card-loader-ui)
